src/ressources/commonFields/FullNameField.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box, Typography } from "@mui/material";
+import { useRecordContext, UseRecordContextParams } from "react-admin";
+import { AvatarField } from "../customers/AvatarField";
+
+export const FullNameField = (props: UseRecordContextParams) => {
+  const record = useRecordContext(props);
+
+  return record ? (
+    <Box display="flex" alignItems="center">
+      <AvatarField record={record} />
+      <Typography ml={10} color="#4f3cc9">
```

Comment:
It's not fundamentally wrong, but just so you know, I used this in my code instead: `<Typography variant="body2">`

Also (this is really a detail I agree), I noticed the avatar and the label are sometimes a bit far away from each other in your screens. I believe the use of `display="flex"` and `alignItems="center"` is not very indicated here.
In case you're interested, in my implementation I used:
```tsx
    <Stack direction="row" spacing={2}>
      <Avatar [...] />
      <Typography variant="body2">
        {customer.first_name} {customer.last_name}
      </Typography>
    </Stack>
```

Reply:
The alignItems property center items in vertical way when the flex direction is row. The problem that you saw was because of the ml={10}, it's relative when we used Mui. 
It's a regression of my code because in an older version, it was a 'span' and the 10 was meaning 10px.
Thank you for the tip!